File: act3/People/Person.py
class Person:
    max_health = 100
    health = 100
    name = ""
    
    def __init__(self, name):
        self.name = name
    
    def is_alive(self):
        if(self.health > 0):
            return True
        return False
    
    def eat(self, food):
        self.health += food.health
        if(self.health >= self.max_health):
            self.health = self.max_health

File: act5/classes/Person.php
<?php

	require_once "interfaces/Human.php";

	abstract class Person implements Human{
		public $name;
		public $health = 100;
		public $max_health = 100;
		
		public function getName(){
			return $this->name;
		}
		
		public function is_alive(){
			if($this->health > 0)
			{
				return true;
			}
			return false;
		}
	}
